ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_CppSamples/ArcGISRuntimeSDKQt_CppSamples.pro

```diff
@@ -196,7 +187,32 @@ SOURCES += \
     $$COMMONVIEWER/SearchFilterSimpleKeywordCriteria.cpp \
     $$COMMONVIEWER/SourceCode.cpp \
     $$COMMONVIEWER/SourceCodeListModel.cpp \
-    $$COMMONVIEWER/mainSample.cpp
+    $$COMMONVIEWER/mainSample.cpp \
+    $$files(zlib-ng/*.c) \
+    $$files(minizip-ng/mz_compat.c) \
+    $$files(minizip-ng/mz_crypt.c) \
+    $$files(minizip-ng/mz_os.c) \
+    $$files(minizip-ng/mz_strm.c) \
+    $$files(minizip-ng/mz_strm_mem.c) \
+    $$files(minizip-ng/mz_strm_zlib.c) \
+    $$files(minizip-ng/mz_zip.c) \
```

Comment:
I would change all these to $$COMMONVIEWER/minizip-ng/filename.c. "files" is more of a wildcarding tool that doesn't make sense when specifying a single file.